app/jobs/whitehall_document_import_job.rb

```diff
@@ -19,6 +19,14 @@ def perform(document_import)
   end
 
   def self.handle_error(document_import, error)
+    if document_import.pending?
+      begin
+        GdsApi.whitehall_export.unlock_document(document_import.whitehall_document_id)
+      rescue StandardError => e
+        logger.warn(e.inspect)
```

Comment:
Sorry to be a pain - may want to prefix this with an intro line `logger.warn("Failed to unlock whitehall document: #{e.inspect}")`

Reply:
Done 👍